docs-devsite/vertexai-preview.generationconfig.md

```diff
@@ -26,7 +26,8 @@ export interface GenerationConfig
 |  [frequencyPenalty](./vertexai-preview.generationconfig.md#generationconfigfrequencypenalty) | number |  |
 |  [maxOutputTokens](./vertexai-preview.generationconfig.md#generationconfigmaxoutputtokens) | number |  |
 |  [presencePenalty](./vertexai-preview.generationconfig.md#generationconfigpresencepenalty) | number |  |
-|  [responseMimeType](./vertexai-preview.generationconfig.md#generationconfigresponsemimetype) | string | Output response mimetype of the generated candidate text. Supported mimetypes are <code>text/plain</code> (default, text output) and <code>application/json</code> (JSON response in the candidates). The model needs to be prompted to output the appropriate response type, otherwise the behavior is undefined. This is a preview feature. |
+|  [responseMimeType](./vertexai-preview.generationconfig.md#generationconfigresponsemimetype) | string | Output response mimetype of the generated candidate text. Supported mimetypes are <code>text/plain</code> (default, text output) and <code>application/json</code> (JSON response in the candidates). |
+|  [responseSchema](./vertexai-preview.generationconfig.md#generationconfigresponseschema) | [TypedSchema](./vertexai-preview.md#typedschema) \| [SchemaRequest](./vertexai-preview.schemarequest.md#schemarequest_interface) | Output response schema of the generated candidate text. This value can be a class generated with a [Schema](./vertexai-preview.schema.md#schema_class) static method like <code>Schema.string()</code> or <code>Schema.object()</code> or it can be a plain JS object matching the [SchemaRequest](./vertexai-preview.schemarequest.md#schemarequest_interface) interface. Note: This only applies when the specified <code>responseMIMEType</code> supports a schema; currently this is limited to <code>application/json</code>. |
```

Comment:
NOT BLOCKING

Can the "Note: ..." be on its own line?

(comment applies to several other instances, even in other files)

Reply:
It's a little weird, I have to put a `<br />` into the code comment, but it works.